numpy/distutils/system_info.py

```diff
@@ -3107,8 +3105,6 @@ def show_all(argv=None):
             del show_only[show_only.index(name)]
         conf = c()
         conf.verbosity = 2
-        # FIXME: r not used
-        r = conf.get_info()
```

Comment:
I suspect this has side effects based on the verbosity settings above, so the `conf.get_info` call is still needed

Reply:
sure, I'll reflect that change in a new commit.